rst_files/lake_model.rst

```diff
@@ -514,12 +485,13 @@ Let's plot the path of the sample averages over 5,000 periods
     using QuantEcon, Roots, Random
 
     Random.seed!(42)
```

Comment:
Moved to the test block as well as the `using Test, Random`

Reply:
Just to be clear, there wasn't an issue with _ever_ seeding the RNG. The issue is just that we don't want that thing to pop up before every block. So I think we can leave this as is.

Reply:
If used in the lectures then it's fine. Just split the `using` statement and the seeding can go right before the execution cell to ensure it gets seeded when it should.

Reply:
Roger that.